bridge/bridge/app.py

```diff
@@ -96,6 +101,19 @@ def main():
             )
             deploy_client.delete_versions(expired_versions)
             logger.info("Update complete.")
+
+            # Analytics
+            if num_new_versions := len(new_versions_map) > 0:
```

Comment:
IMO it is not very useful to track the derivative like this. I think just tracking the number of models managed is sufficient. 

The only metric like this that I would want to see is how many times a stage changed model version, to get an idea of how useful this is to them.